spec/jobs/sync_topic_user_bookmarked_spec.rb

```diff
@@ -39,16 +39,10 @@
     expect(tu2.reload.bookmarked).to eq(false)
   end
 
-  it "works when no topic id is provided (runs for all topics)" do
```

Comment:
Removed this because there is nowhere in the codebase that needs to run this for all topics; this would be super slow as well.

Reply:
> there is nowhere in the codebase that needs to run this for all topics

Should the job `throw` if the arg is not provided?

Reply:
Yep, will add